### DISCUSSION TITLE
Anyone know how to add Maria Db 10.6?

### DISCUSSION THREAD

**Discussion by cretace** [leokhoa/laragon#618] [Q&A] [answered]
Adding an older versions of maria DB is easy
In packages.conf (under Tools: Quick add : Configuration) I just add:
mariadb10.3=https://downloads.mariadb.org/f/mariadb-10.3.10/winx64-packages/mariadb-10.3.10-winx64.zip/from/http%3A//sgp1.mirrors.digitalocean.com/mariadb/?serve

BUT there's no comparable link to version 10.6
There's only the MariaDB download page and it's all hidden under buttons that must trigger Javascript.

This of course is for Windows - there's links to Linux versions but nothing for Windows
Any ideas?

**Comment by cretace** [comment]
Well I seem to have added it ok by unzipping it into the Laragon Mysql directory
BUT while Laragon _appears_ to switch to MariaDb any php code shows it's still using Mysql 8, in phpMyAdmin or PHP $db-> server_info.

**Reply by RegalChivas** [reply, marked as answer]
try this
`mariadb_11.2.2=https://archive.mariadb.org//mariadb-11.2.2/winx64-packages/mariadb-11.2.2-winx64.zip`
`mariadb_10.6.16=https://archive.mariadb.org//mariadb-10.6.16/winx64-packages/mariadb-10.6.16-winx64.zip`

**Comment by cretace** [comment]
Great thanks!! 
Why didn't I spot the mirror part of the URL could be removed, and why do MariaDb make it impossible to find  :-(
